lib/src/picture_provider.dart

```diff
@@ -436,7 +440,7 @@ class PictureKey<T> {
   final ColorFilter? colorFilter;
 
   /// The theme used when this key was created.
-  final SvgTheme theme;
+  final SvgTheme? theme;
```

Comment:
As the theme is nullable, would this be okay to make this property null in PictureKey as well?

Reply:
Reversed to be non-nullable.